daprdocs/content/en/reference/components-reference/supported-state-stores/setup-dynamodb.md

```diff
@@ -70,6 +73,39 @@ In order to use DynamoDB TTL feature, you must enable TTL on your table and defi
 The attribute name must be defined in the `ttlAttributeName` field.
 See official [AWS docs](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/TTL.html).
 
+## Partition Keys
+
+The DynamoDB state store will use the `key` property provided in the requests to the Dapr API to determine the partition key. This can be overridden by specifying a metadata field in the request with a key of `partitionKey` and a value of the desired partition.
```

Comment:
I would also suggest providing a link to this relevant doc. https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.CoreComponents.html#HowItWorks.CoreComponents.PrimaryKey 
Confirm this is the best link. 

I would also suggest that you change the example below to use the PersonID table in the AWS example above rather than Tony Stark. That way people can related the two example better and you are aligned with AWS documentation,